android/app/build.gradle

```diff
@@ -26,7 +26,7 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 apply from: project(':flutter_config').projectDir.getPath() + "/dotenv.gradle"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
so should I revert these changes ?, cuz even I was using flutter 2,.10, so just thought it would be a good idea to do it all in one.
@ItsAdityaKSingh

Reply:
Yup. Those changes are required for successful execution, but they have already been looked into, so you would want to keep those changes to your local system only. I hope you get my point here :)